pipeline.go

```diff
@@ -95,7 +96,15 @@ func clearSetup() error {
 	return nil
 }
 
-// Run executes the pipeline.
+// Run executes the pipeline. For each stage defined in the pipeline we execute
+// the `docker build` and `docker run`. If any of these two processes fail,
+// we interrupt the flow and the error handler is called. The error handler is a
+// stage like any other, but it is only built and executed in case of an error in
```

Comment:
Ele é ligeiramente diferente pois é invocado com uma STDIN especial, certo?

Reply:
Tinha colocado que ele era construído e executado em caso de erro no fluxo padrão do pipeline.

Tentei ajustar para o seguinte "The error handler can be defined as a stage, but it will only be executed in case of an error in the standard pipeline flow."